iBudget/spending/views.py

```diff
@@ -198,14 +198,11 @@ def create_spending_category(request):
         return HttpResponse("Bad request", status=400)
     spending = SpendingCategories.filter_by_owner_name(owner=owner, name=name)
 
-    if not spending:
-        spending = SpendingCategories(name=name, icon=icon, owner=owner, is_shared=is_shared)
-        try:
-            spending.save()
-        except(ValueError, AttributeError):
-            return HttpResponse(status=406)
-    else:
+    if spending:
```

Comment:
What is the reason to change this check?

Reply:
The reason is Yevgen's disagreement